rpc/types_transaction_response.go

```diff
@@ -14,6 +14,12 @@ type AddDeployAccountTransactionResponse struct {
 	ContractAddress *felt.Felt `json:"contract_address"`
 }
 
+// AddDeployTransactionResponse provides the output for AddDeployTransaction.
+type AddDeployTransactionResponse struct {
+	TransactionHash *felt.Felt `json:"transaction_hash"`
+	ContractAddress *felt.Felt `json:"contract_address"`
+}
```

Comment:
This type looks exactly the same as `AddDeployAccountTransactionResponse`, and I don't see it being used anywhere on the code. What's the point?

Also both types have the same comment on the top (probably because of copy-pasting), if they are meant for different things it 'll be good to change the comments

Reply:
This is no longer needed, I'll remove it